### PR TITLE
[11.0][mig][pos_session_pay_invoice]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Migration to v11
POS Session Pay invoice
=======================

This modules allows to pay an existing Supplier Invoice / Customer Refund, or
to collect payment for an existing Customer Invoice, from within a POS Session.

Installation
============

* In order to install this module, you will need to install first the
  module 'account_cash_invoice', available in https://github.com/OCA/account-payment/11.0

Configuration
=============
#.  Go to *Point of Sale / Configuration / Point of Sale* and activate the
    'Cash Control' setting.

Usage
=====

#.  Go to *Point of Sale / Dashboard* and create and open or access to an
    already open POS Session.
#.  Press the button **Pay Invoice** to pay a Supplier Invoice or a Customer
    Refund. It will be paid using Cash.
#.  Select **Collect Payment from Invoice** in to receive a payment of an
    existing Customer Invoice or a Supplier Refund. You will need to select
    a Journal if the POS Config has defined multiple Payment Methods.

Known issues / Roadmap
======================

* Cannot pay invoices in a different currency than that defined in the journal
  associated to the payment method used to pay/collect payment.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
cc @rgarnau @etobella @mreficent

File: pos_session_pay_invoice/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+========================
+
+This modules allows to pay an existing Supplier Invoice / Customer Refund, or
+to collect payment for an existing Customer Invoice, using bank statements.
+
+
+Usage
+=====
+
+#.  Go to *Accounting / Dashboard* and create and/or open an existing
```
Comment (rgarnau) [inline pos_session_pay_invoice/README.rst]: Instead of 'Accounting' it should be 'Invoicing/Accounting'

**Review by rgarnau — APPROVED** [review]
LGTM :+1:

**Comment by JordiBForgeFlow** [comment]
@legalsylvain Can you review this one?

**Review by etobella — APPROVED** [review]
LGTM :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
Nice, but I'm not a fan to include the setup files...

**Pull request merged**